pasqal-cloud/pasqal_cloud/__init__.py

```diff
@@ -176,6 +176,16 @@ def _validate_device_type(
         emulator: Optional[EmulatorType],
         device_type: Optional[DeviceTypeName],
     ) -> DeviceTypeName:
+        if device_type == DeviceTypeName.EMU_TN:
```

Comment:
```suggestion
        if device_type == DeviceTypeName.EMU_TN or emulator == EmulatorType.EMU_TN:
```

Otherwise, users using the EmuTNBackend or the emulator attribute won't see this warning message